commit f651ac2160b24f8e08f5a18c583cb1bd38c3b248
Author: Jiří Klimeš <jklimes@redhat.com>
Date:   Wed May 23 10:50:28 2012 +0200

    dbus: remove 'max_replies_per_connection' limit from D-Bus configuration
    
    It is leftover from times when D-Bus default limit was 32. Now, it is 8192, see
    http://cgit.freedesktop.org/dbus/dbus/commit/?id=8d3d8ff55739eebd84d0d53a20a025329feafc3b

diff --git a/org.freedesktop.ModemManager.conf.nopolkit b/org.freedesktop.ModemManager.conf.nopolkit
index 2f33161..0dd5ee7 100644
--- a/org.freedesktop.ModemManager.conf.nopolkit
+++ b/org.freedesktop.ModemManager.conf.nopolkit
@@ -11,7 +11,5 @@
   <policy user="root">
     <allow own="org.freedesktop.ModemManager"/>
   </policy>
-
-  <limit name="max_replies_per_connection">512</limit>
 </busconfig>
 
diff --git a/org.freedesktop.ModemManager.conf.polkit b/org.freedesktop.ModemManager.conf.polkit
index 25490e3..768b2ad 100644
--- a/org.freedesktop.ModemManager.conf.polkit
+++ b/org.freedesktop.ModemManager.conf.polkit
@@ -148,7 +148,5 @@
     <allow own="org.freedesktop.ModemManager"/>
     <allow send_destination="org.freedesktop.ModemManager"/>
   </policy>
-
-  <limit name="max_replies_per_connection">512</limit>
 </busconfig>
 
